src/DeepCopy/DeepCopy.php

```diff
@@ -85,11 +85,33 @@ public function copy($value)
         return $this->recursiveCopy($value);
     }
 
+    /**
+     * @param Filter  $filter
+     * @param Matcher $matcher
+     *
+     * @return void
+     */
     public function addFilter(Filter $filter, Matcher $matcher): void
     {
         $this->filters[] = [$matcher, $filter];
     }
 
+    /**
+     * Add filter with high priority
+     *
+     * @param Filter  $filter
```

Comment:
Same here. I think the method is self-describing enough to skip the description `Add filter with high priority`, but I don't mind leaving it either.

Reply:
OK